flexmeasures/ui/templates/base.html

```diff
@@ -51,79 +51,87 @@
     {% block body %}
 
     {% block nav %}
-
-    <nav class="navbar navbar-default navbar-fixed-top" id="topnavbar">
-        <div id="navbar-container" class="container-fluid">
-            <div class="navbar-header">
-                <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
```

Comment:
Why is this button gone? Was it unused?
(I have trouble remembering what is or was)

Reply:
it was for the hamburger, the navbar-toggler replaced it here,

Reply:
solved